__tests__/integration/configuration/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       "--allow-mining-without-peers",
       "--unsafe-mode", "nano-testnet-alpha",
       "--data", "./tmp",
-      "--nc-history-index",
```

Comment:
Why?

Reply:
The parameter changed on hathor-core. Now we have `--nc-indices` for all nano contract indexes 